src/prognostic_equations/forcing/nudging.jl

```diff
@@ -0,0 +1,63 @@
+#####
+##### Nudging/relaxation tendency for GCM-driven SCM
+#####
+
+import ClimaCore.Fields as Fields
+
+nudging_cache(Y, atmos::AtmosModel) = nudging_cache(Y, atmos.nudging)
+
+nudging_cache(Y, ::Nothing) = (;)
+nudging_tendency!(Yₜ, Y, p, t, colidx, ::Nothing) = nothing
+
+function nudging_cache(Y, ::Nudging)
+    FT = Spaces.undertype(axes(Y.c))
+    ᶜT_mean = similar(Y.c, FT)
+    ᶜq_tot_mean = similar(Y.c, FT)
+    ᶜu_mean = similar(Y.c, FT)
+    ᶜv_mean = similar(Y.c, FT)
+    # TODO: read profiles from LES
+    @. ᶜT_mean = 290
+    @. ᶜq_tot_mean = 0.010
+    @. ᶜu_mean = 5
+    @. ᶜv_mean = 0
+    # TODO: make it a function of z and add timescale to climaparams
+    hr = 3600
+    ᶜτ_scalar = 24hr
+    ᶜτ_wind = 6hr
```

Comment:
Maybe use a coefficient rather than timescale so that we can have 0 relaxation for scalar or wind separately (by setting the coefficient to 0) without an additional flag.